### DISCUSSION TITLE
¿Cómo me uno al grupo de Teams de la misión Backend?

### DISCUSSION THREAD

**Discussion by Alexl099** [Launch-X-Latam/MisionFrontEnd#485] [Team Posts]
Hola compañeros explorers! Podrian indicarme como unirme al grupo de teams de la mision backend? Ya complete el formulario que brindaron, no me llega nada aun. Agradecería mucho su ayuda compañeros. Gracias por su atención.

**Comment by Oswald22a** [comment]
Hola compañero!, en mi caso recibí un enlace para unirme en mi correo personal, te recomiendo revisar tu bandeja de entrada o spam del correo que proporcionaste en el registro. Igual si alguien más tiene otra manera en que se pudo unir y nos la pueda compartir.
Saludos!

**Comment by Kariid26** [comment]
Hola, tengo una duda respecto al tema, cuando abro el enlace me aparece la opción “unirse al grupo” de la misión backend pero cuando lo selecciono no sucede nada, es automático unirse al grupo o deben aceptar mi solicitud?

**Comment by Oswald22a** [comment]
Hola, yo también tuve ese problema, al darle clic en unirte solo se quedaba cargando la aplicación pero no hacía nada, sin embargo al día siguiente ya me apareció que me habían añadido al grupo, desconozco si tendrán que aceptar la solicitud para unirnos.
Si Justo hoy intentaste unirte te recomiendo esperar unas horas para validar si ya apareces añadida al grupo

**Comment by Kariid26** [comment]
Muchas gracias, justo lo intente hace una media hora, entonces esperaré y volveré a revisar más tarde. Gracias por la ayuda😊

Obtener Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: Oswald22a ***@***.***>
Enviado: Monday, November 7, 2022 1:00:12 AM
CC: Aída Karina Ibarra ***@***.***>; Comment ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] ¿Cómo me uno al grupo de Teams de la misión Backend? (#11)


Hola, yo también tuve ese problema, al darle clic en unirte solo se quedaba cargando la aplicación pero no hacía nada, sin embargo al día siguiente ya me apareció que me habían añadido al grupo, desconozco si tendrán que aceptar la solicitud para unirnos.
Si Justo hoy intentaste unirte te recomiendo esperar unas horas para validar si ya apareces añadida al grupo

**Comment by pieropr2k** [comment]
A mi me sale "Vaya... no se puede encontrar ese equipo
Parece que el equipo o el canal que busca no existe. Póngase en contacto con el propietario del equipo o vuelva a intentarlo."

**Comment by Alexl099** [comment]
Gracias por su ayuda explorers, ya me llegó el correo. Al parecer gmail habia almacenado el mensaje en otra parte. Mil gracias a todos por su apoyo.
